core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MultiregionalBucketMigrationActor.scala

```diff
@@ -598,7 +598,8 @@ object MultiregionalBucketMigrationActor {
             .updateBucketIam(
               bucket,
               bucketPolicies,
-              Option.when(migration.requesterPaysEnabled)(GoogleProjectId(deps.googleProjectToBill.value))
+              Option.when(migration.requesterPaysEnabled)(GoogleProjectId(deps.googleProjectToBill.value)),
+              iamPolicyVersion = 3
```

Comment:
What version would it have been at, in the case of a FastPass grant, if you didn't specify 3?

Reply:
The policy we are setting will default to 1 since there are no conditionals. The existing policy on the bucket is 3 because of the FastPass grants.

Reply:
And there is no harm in setting it to 3 for all buckets?

Reply:
Not that I can see. It's actually recommended by Google and it's the policy version we're using if a FastPass grant has been issued.